src/test/java/com/imgix/test/TestAll.java

```diff
@@ -63,40 +69,54 @@ public void testSetUseHttpsTrue() {
     @Test
     public void testHelperBuildAbsolutePath() {
         URLHelper uh = new URLHelper("securejackangers.imgix.net", "/example/chester.png", "http");
-        assertEquals(uh.getURL(), "http://securejackangers.imgix.net/example/chester.png");
+        assertEquals("http://securejackangers.imgix.net/example/chester.png", uh.getURL());
```

Comment:
Sonar-lint was mad that expected vs actual were in the wrong place here. Had to swap them in several places.